client/components/boards/boardsList.jade

```diff
@@ -26,8 +26,6 @@ template(name="boardList")
 
 
 template(name="boardListHeaderBar")
-  h1 {{_ 'my-boards'}}
```

Comment:
I removed the title intentionally for purpose of simplicity. As there is already a logo as the title, the text looks a little superfluous on the main page, especially on the header of small screen.